app/controllers/songs_controller.rb

```diff
@@ -0,0 +1,58 @@
+class SongsController < ApplicationController
+  before_action :set_song, only: [:show, :edit, :update, :destroy]
+
+  # GET /songs
+  def index
+    @songs = Song.all
+  end
+
+  # GET /songs/1
+  def show
+  end
+
+  # GET /songs/new
+  def new
+    @song = Song.new
+  end
+
+  # GET /songs/1/edit
+  def edit
+  end
+
+  # POST /songs
+  def create
+    @song = Song.new(song_params)
+
+    if @song.save
+      redirect_to @song, notice: 'Song was successfully created.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.